example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -509,9 +509,12 @@ describe('The S3 Ingest Granules workflow', () => {
     it('sets granule.createdAt with value from SyncGranule', async () => {
       failOnSetupError([beforeAllError, subTestSetupError]);
 
-      await waitForModelStatus(
-        granuleModel,
-        { granuleId: lambdaOutput.meta.input_granules[0].granuleId },
+      await waitForApiStatus(
```

Comment:
Was this broken in master?

Reply:
The opposite, I think?

We've removed the use of `waitForModelStatus` from this integration test in the feature branch, since Dynamo is no longer the source of the truth and we should be going through the API

https://github.com/nasa/cumulus/blob/feature/rds-phase-2/example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

Reply:
Sure, but even with Dynamo being the source of truth we shouldn't be using a direct model call, I was curious if the direct-model-query pattern was used recently.

Reply:
We still have a lot of places we're using `waitForModelStatus`

https://github.com/nasa/cumulus/search?q=waitForModelStatus

To be honest it's something we probably need to address in the feature branch that isn't ticketed